pkg/cidata/cidata.TEMPLATE.d/boot/11-colorterm-environment.sh

```diff
@@ -1,18 +1,24 @@
 #!/bin/sh
 set -eux
 
-# openSUSE Tumbleweed uses /etc/ssh/sshd_config.d, not /etc/ssh/sshd_config
-# TODO: support /etc/ssh/sshd_config.d
-if [ ! -e /etc/ssh/sshd_config ]; then
-	exit 0
+if [ -d /etc/ssh/sshd_config.d ]; then
+	if [ -e /etc/ssh/sshd_config.d/10-acceptenv-colorterm.conf ]; then
+		exit 0
+	fi
+
+	# accept any incoming COLORTERM environment variable
+	echo "AcceptEnv COLORTERM" >/etc/ssh/sshd_config.d/10-acceptenv-colorterm.conf
 fi
 
-if grep -q "COLORTERM" /etc/ssh/sshd_config; then
-	exit 0
+if [ -e /etc/ssh/sshd_config ]; then
```

Comment:
Should this be an `elif`? Otherwise you are creating the separate file **and** also patching the config. I guess it doesn't hurt, but wanted to make sure it is intentional.

```suggestion
elif [ -e /etc/ssh/sshd_config ]; then
```

Reply:
I was thinking that it might be overridden in the default sshd_config, after including the files...

Currently it seems to be either/or in the distributions, but could be a possibility in future debian?

Reply:
You can have multiple `AcceptEnv` settings, and they are cumulative:

```
AcceptEnv FOO
AcceptEnv BAR
```

is the same as

```
AcceptEnv FOO BAR
```

Reply:
Ah, OK. I think I will change the code to use `elif` then - and to exit if it doesn't find anything (shouldn't happen)